README.md

```diff
@@ -1,2 +1,24 @@
-# city-sr-api-example
-Example code and documentation for talking to the City's Service Request API
+# CCT CUSTOMER SERVICE REQUESTS
+This section provides a descriptive overview of the concepts and their implementation and of the resources and their use.
+## API Description
+## HTTP Header Parameters
+The header parameters used in this API are defined in:
+
+| Parameter Name  |Description   | Value characteristics  | 
+|---|---|---|
+| X-Service  | This is the value of the public key issued to the registered consumer by the City. Must be provided as a mandatory parameter in all messages, |  GUID of 32 characters with no punctuation |   
+| X-Session    | This is the session key obtained periodically from the City representing the log file entry of a user’s activities. The session identifier must be provided in all calls other than the service that returns the session identifier. |  GUID of 32 characters with no punctuation.  |    
+| X-Signature  | This is a hexadecimal string containing a key-hashed message authentication code (HMAC) calculated from the message string using the SHA256 hash algorithm and encoded using base 64.   | Base 64 encoded string containing an HMAC SHA256 authentication code.  |  
+
+## OpenAPI 3.0 Guide
```

Comment:
I must admit I was a bit dubious about this, but I like this section - it gives a high level overview of the spec format.
 
Maybe also link to the OpenAPI docs, as well as some a good getting started guide.

Reply:
I was more thinking [the official OpenAPI spec](https://swagger.io/specification/)